pom.xml

```diff
@@ -339,6 +339,13 @@
             <version>${dd-java-agent-version}</version>
             <scope>test</scope>
         </dependency>
+        <!--
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-jdk14</artifactId>
+            <version>1.7.25</version>
+        </dependency>
+        -->
```

Comment:
Do we still need this ^?

Reply:
I am hesitant to enable it for tests only.